albatross/core/model.h

```diff
@@ -92,6 +92,24 @@ template <typename ModelType> class ModelBase : public ParameterHandlingMixin {
   }
 
 public:
+  template <class Archive> void save(Archive &archive) const {
```

Comment:
This is probably a cereal thing, but it's not immediately obvious what the difference is between `save()` and `serialize()` below.

Reply:
Yeah ... that's a cereal thing.  https://uscilab.github.io/cereal/serialization_functions.html

Basically, if you can get away with directly serializing all members of an object you can usually just define `serialize`, but if you need to go through `get_` and `set_` methods then you end up needing `save` and `load`.